src/classlibnative/bcltype/number.cpp

```diff
@@ -134,8 +135,191 @@ unsigned int Int64DivMod1E9(unsigned __int64* value)
 
 #else // _TARGET_X86_ && !FEATURE_PAL
 
-#pragma warning(disable:4273)
-extern "C" char* __cdecl _ecvt(double, int, int*, int*);
+void ecvt( double value, int count, int* dec, int* sign, wchar_t* digits )
```

Comment:
Also, regardless of which route you decide. It would probably be good to remove the old _ecvt implementation (defined in the PAL layer) anyways, as it is dead code now.